packages/react-examples/src/react-tooltip/Tooltip/Tooltip.stories.tsx

```diff
@@ -1,58 +1,169 @@
 import * as React from 'react';
-import { Tooltip, TooltipImperativeHandle } from '@fluentui/react-tooltip';
-import { ThemeProvider } from '@fluentui/react-theme-provider';
+import { Tooltip } from '@fluentui/react-tooltip';
 import { webLightTheme } from '@fluentui/react-theme';
 import { makeStyles } from '@fluentui/react-make-styles';
+import { FluentProvider } from '@fluentui/react-provider';
 
 const useStyles = makeStyles({
+  exampleList: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+    margin: '16px 0',
+    gap: '16px',
+  },
+
+  targetContainer: {
+    display: 'inline-grid',
+    alignItems: 'center',
+    justifyItems: 'center',
+    gap: '4px',
+    margin: '16px 128px',
+  },
+
   target: {
-    width: '128px',
-    height: '128px',
-    margin: '64px',
-    background: 'lightgray',
-    border: '1px solid darkgray',
+    width: '48px',
+    height: '48px',
   },
 });
 
-export const TooltipExample = () => (
-  <ThemeProvider theme={webLightTheme}>
-    <TooltipExampleCore />
-  </ThemeProvider>
-);
+export const Basic = () => {
+  const styles = useStyles();
+
+  const targetRef1 = React.useRef<HTMLDivElement>(null);
```

Comment:
```suggestion
  const targetRef = React.useRef<HTMLDivElement>(null);
```

_nit_, this can be just `targetRef`